Functions/FSL/xASL_fsl_RunFSL.m

```diff
@@ -85,44 +85,48 @@
 %     ExistCUDA = false;
 % end
 
+% Setup VABY quantification
```

Comment:
Well, I hope you understand that this is actually quick and dirty ;)

xASL_fsl_RunFSL is a script that prepares for running FSL, what are the settings that we should give FSL, what is the Bash initialization etc. So it is 90% for FSL specifically, so if you then use the 10% that is more generic for VABY that we will only use once that is asking for diluted code :)

I propose instead:
1. rename xASL_fsl_RunFSL -> xASL_run_FSL
2. create xASL_run_VABY

Reply:
I am aware of all these shortcomings. So alerted about that in the issue ;)

My main point was that I need to quickly set up the functionality. And then I'll distribute the functionality across the correct files as we agree together.

I agree with your proposition, but we need to adapt the code as we have an FSL directory in Functions.

I propose to create Functions/External
then have xASL_ext_runFSL, xASL_ext_runVABY, xASL_ext_SetFSLdir, xASL_ext_TopUpFSL

Agree like that, or other names?

Reply:
Yes that sounds good, though I would do xASL_fsl_TopUp xASL_vaby_Quant xASL_fsl_Setdir etc in the external folder
Or as you have it, but fsl before topup, not topupFSl

So our prefix then (ext if you want it) then the software name then the function name

Reply:
If you don't mind, I would move all to External and use xASL_ext_
The main reason is that we have FSL, VABY - that would be already two directories and could grow up a lot. So best to keep the number of directories in functions reasonable and have all in ext.
So the function names will be

xASL_ext_FSLrun
xasl_ext_VABYrun
xASL_ext_FSLsetDir
xASL_ext_FSLTopUp

OK? The other option is 
Functions/FSL/xASL_fsl_Run
Functions/VABY/xASL_vaby_Run etc...

Reply:
Yes good idea, I only proposed to remove the ext prefix but you're right that it is better.

Reply:
OK. So the final list of changes is
- [ ] Remove FSL directory in functions and create EXT
- [ ] xASL_fsl_RunFSL.m -> xASL_ext_FSLrun
- [ ] xASL_fsl_SetFSLdir.m -> xASL_ext_FSLsetDir
- [ ] xASL_fsl_TopUp.m -> xASL_ext_FSLtopUp
- [ ] Split code to a new function xASL_ext_VABYsetDir
- [ ] Split code to a new function xASL_ext_VABYrun